overlord/ifacestate/ifacestate.go

```diff
@@ -112,8 +112,8 @@ func Connect(st *state.State, plugSnap, plugName, slotSnap, slotName string) (*s
 	return connect(st, nil, nil, plugSnap, plugName, slotSnap, slotName)
 }
 
-func connect(st *state.State, change *state.Change, installedSnapTask *state.Task, plugSnap, plugName, slotSnap, slotName string) (*state.TaskSet, error) {
-	if err := checkConnectConflicts(st, change, plugSnap, slotSnap, installedSnapTask); err != nil {
+func connect(st *state.State, change *state.Change, mainTask *state.Task, plugSnap, plugName, slotSnap, slotName string) (*state.TaskSet, error) {
+	if err := checkConnectConflicts(st, change, plugSnap, slotSnap, mainTask); err != nil {
```

Comment:
we should remember to probably s/mainTask/installedSnapTask/ here too when we un-revert